independent-publisher-connectors/DeepL/apiProperties.json

```diff
@@ -0,0 +1,65 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "environment": {
```

Comment:
@mromiszewski we are getting an error regarding this environment property.
##[error]@{level=Error; message=JSON is valid against no schemas from 'oneOf'.; valuePath=line: 0 position: 0 path: 'properties.connectionParameters.environment'; value=}

May I ask you please to move this property inside of the "connectionParameters" scope.

Reply:
It is odd because I have a similar schema like the existing connector: https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/ServiceObjects/apiProperties.json

Reply:
Hi @mromiszewski, thank you so much for bringing this to our attention. 
May I ask you please to test the connector to make sure that this change did not break the connector?
I know the errors from the apiProperties validator are fixed but just to make sure.
We will approve and move forward once you confirm the testing.
Thank you so much for your hard work with us.

Reply:
I have imported the connector to the new tenant/environment and created a flow with the connector there. Everything works fine.